core/test/fixtures/fraggle-rock/reports/sample-flow-result.json

```diff
@@ -3299,10 +3299,11 @@
           },
           "doctype": {
             "id": "doctype",
-            "title": "Page has the HTML doctype",
+            "title": "Page lacks the HTML doctype, thus triggering quirks-mode",
```

Comment:
Looks like the doctype artifact is out of date on the flow artifacts:
`yarn update:flow-sample-json --rebaseline-artifacts Doctype`

Reply:
yeah thx

btw
<img width="955" alt="image" src="https://user-images.githubusercontent.com/4071474/206012672-a01f39f9-3c1b-49fd-945b-97ee1d904fa5.png">